pkg/controller/gittrackobject/gittrackobject_controller_test.go

```diff
@@ -511,348 +537,474 @@ var _ = Describe("GitTrackObject Suite", func() {
 		Context("with a ClusterGitTrackObject", func() {
 			var gto *farosv1alpha1.ClusterGitTrackObject
 			var child *rbacv1.ClusterRoleBinding
+			var originalUID types.UID
 
-			BeforeEach(func() {
-				gto = testutils.ExampleClusterGitTrackObject.DeepCopy()
-				gto.SetOwnerReferences([]metav1.OwnerReference{
-					{
-						APIVersion: "faros.pusher.com/v1alpha1",
-						Kind:       "GitTrack",
-						UID:        gitTrack.UID,
-						Name:       gitTrack.Name,
-					},
+			Context("and ClusterGitTrackMode is set to Disabled", func() {
+				BeforeEach(func() {
+					SetupTest(farosflags.GTMEnabled, farosflags.CGTMDisabled)
+				})
+				BeforeEach(func() {
+					gto = testutils.ExampleClusterGitTrackObject.DeepCopy()
+					gto.SetOwnerReferences([]metav1.OwnerReference{
+						{
+							APIVersion: "faros.pusher.com/v1alpha1",
+							Kind:       "ClusterGitTrack",
+							UID:        clusterGitTrack.UID,
+							Name:       clusterGitTrack.Name,
+						},
+					})
+					child = testutils.ExampleClusterRoleBinding.DeepCopy()
+					Expect(testutils.SetGitTrackObjectInterfaceSpec(gto, child)).To(Succeed())
+					m.Update(child).Should(Succeed())
+					// get the most recent version so we can get the UID
+					m.Get(child, timeout).Should(Succeed())
+					originalUID = child.GetUID()
+
+					// Create
+					m.Create(gto).Should(Succeed())
+				})
+
+				It("should not get reconcile ClusterGitTrackObjects", func() {
+					Consistently(requests, consistentlyTimeout).ShouldNot(Receive(Equal(expectedClusterRequest)))
+				})
+				It("should not update the child", func() {
```

Comment:
Nit: Should have a new line between `It`s on the same indentation level to help readability